jobs/intg-test-resources/wso2-intg-test-cfn.yaml

```diff
@@ -317,9 +321,18 @@ Resources:
         UserData:
           !Base64
           'Fn::Sub': |
-            <script>
-            setx -m JAVA_HOME "%${JDK}%"
-            </script>
+            <powershell>
+            [Environment]::SetEnvironmentVariable("TEST", "$Env:ORACLE_JDK8", "Machine")
```

Comment:
We no longer need to set the JAVA_HOME now?
What are these parameters? TEST, $ENV:ORACLE_JDK8 etc.?

Reply:
Added by mistake. Updated the to set the JAVA_HOME.

Reply:
I'm still confused. We have hard-coded the jdk to ORACLE_JDK8. The actual input parameter in $JDK, isn't it?

Reply:
Fixed and tested.